references/cheat-sheet.md

```diff
@@ -0,0 +1,22 @@
+# Cheat Sheet
+
+## Working with stacks and services
+
+    # Stop all services
+    $ kontena service ls -q | xargs -L1 kontena service stop --force
+
+    # Connecting to a specific instance
+    $ kontena service exec --instance NUM servicename
+
+## Working with kontena components
+
+    # See the kontena/lb config
+    $ kontena service exec ingress-lb/lb cat /etc/haproxy/haproxy.cfg
+
+## Administrative tasks
+
+    # Restart first 3 agents
+    $ for i in `seq 1 3`; do kontena container exec -it node-$i.grid.net/kontena-agent kill 1; done
```

Comment:
This assumes special naming is used on the nodes. Maybe add some comment on that.

Reply:
As above, I'm giving full rights to modify the text so that it follows the Tone of Kontena Docs. I think some level of bash skills should be expected from the reader so that she/he just doesn't blindly copy-paste. Inspiration is more important here.